core/lib/src/fairing/fairings.rs

```diff
@@ -84,7 +84,7 @@ impl Fairings {
                 .collect();
 
             // Reverse the dup indices so `remove` is stable given shifts.
-            dups.sort(); dups.dedup(); dups.reverse();
+            dups.sort_unstable(); dups.dedup(); dups.reverse();
```

Comment:
We want a stable sort. It's even in the comment above. :)

Reply:
`dups` is a vector of `usize`. Two `usize` values compare equal only if they are bitwise equal, so `sort_unstable` produces the same result as `sort`. So I think that clippy is not wrong here.